pkg/erctl/cmd/show.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var showCmd = &cobra.Command{
+	Use:   "show",
+	Short: "specify which filed you want show",
+	Long: `all struct field has been set 'json:",omitempty"', so the programme will set those field null value
```

Comment:
Delete "all struct field has been set 'json:",omitempty"', so the programme will set those field null value"